k8sfv/run-test

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash -ex
+
+# Run the 'k8sfv' test suite, a functional and scale test of Felix
+# running with the Kubernetes datastore driver, driven by CRUD
+# operations through the k8s API server.
+
+# Config.
+#
+# What version of the k8s API server to use.
+: ${K8S_VERSION:=1.5.3}
+#
+# A string to insert into the container names that we use; a calling
+# script can set this to something to make the container names unique,
+# and hence to allow multiple copies of this script to run in
+# parallel.
+: ${UNIQUE:=}
+#
+# URL of a Prometheus push gateway that k8sfv should push metrics to.
+: ${PROMPG_URL:=}
+#
+# Code level to report for the metrics that this test run generates.
+# A CI job that regularly tests a particular (checked in) code branch
+# should set this to the name of that branch, e.g. 'master'.
+: ${CODE_LEVEL:=dev}
+
+# Generate a unique prefix for the names of the containers that we
+# will create.
+prefix=k8sfv${UNIQUE}
+
+# Arrange to always clean up the containers that we create.
+function cleanup {
+    # Clean up the containers that this script generates.
+    docker rm -f ${prefix}-felix || true
+    docker rm -f ${prefix}-apiserver || true
+    docker rm -f ${prefix}-etcd || true
+    # List all remaining containers.
+    docker ps -a
+}
+trap cleanup EXIT SIGINT SIGQUIT
```

Comment:
Nice; I suggest making it configurable whether we run cleanup.  I've certainly found it useful to have the option to leave the rig running when I'm debugging the other FV frameworks.  WDYT?